product-service/api/index.ts

```diff
@@ -0,0 +1,8 @@
+import { Product } from '../models';
+import products from './productList.json';
+
+export const getProductList = async (): Promise<Array<Product>> => Promise.resolve(products);
+export const getProductById = async (queryId: string): Promise<Product> => new Promise((resolve, reject) => {
+    const product = products.find(({id}) => id === queryId);
+    product ? resolve(product) : reject({message: 'Product not found!', error: new Error('Product not found!')});
```

Comment:
For interoperability with other exceptions it is better to reject with an instanceof Error instead of plain object (`reject(new Error(...))`)

Reply:
I've checked it and due to some reason error object is empty on AWS. Still don't know why

Reply:
i.e. no trace, no message